experiments/browser_based_querying/src/components/DocExplorer.tsx

```diff
@@ -125,9 +125,7 @@ export function DocExplorer(props: DocExplorerProps) {
           )}
         </Grid>
         <Grid item container direction="column">
-          <Typography>
-            {content}
```

Comment:
This is what fixes the font issue though :(
What error was this causing?

Reply:
Oh is this the `p` problem? You can fix it by doing `<Typography component="div">`, that way it doesn't use a `p` tag.

Reply:
This is the `p` warning/error that appears in the log. What's the rationale for including a <Typography> node at this level? `{content}` always comes wrapped in an appropriate node?

Reply:
It turns out it doesn't, sometimes it's unwrapped and results in unstyled text. Putting this at the top level ensures we always use the correct font styles no matter what we render!

Reply:
Ah makes sense